adapters/repos/db/index.go

```diff
@@ -1723,17 +1714,5 @@ func (i *Index) addNewShard(ctx context.Context,
 }
 
 func (i *Index) validateMultiTenancy(tenantKey string, object *models.Object) error {
```

Comment:
I would use `Index.PartioningEnabled` (see comment above) to avoid quering sharding state

Reply:
done